docs/templating/layout/customize_storefront_layout.md

```diff
@@ -67,6 +67,131 @@ The most important templates are:
 |`cart/component/add_to_cart/add_to_cart.html.twig`|"add to cart" element|
 |`cart/component/summary/summary.html.twig`|cart summary|
 
+#### Extend Twig template
+
+```html+twig
+{% extends '@IbexaCart/themes/standard/cart/component/minicart/minicart.html.twig' %}
+
+{% block content %}
+    <img
+        class="ibexa-store-minicart__image"
+        src="{{ asset('bundles/ibexastorefront/img/icons/cart.svg') }}"
+        alt="{{ 'minicart.icon.alt'|trans|desc('Cart') }}"
+    />
+    {{ parent() }}
+{% endblock %}
+```
+
+To avoid self-reference, `@IbexaCart` is used instead of `@ibexadesign`.
+
+Built-in components are not styled, so you can freely customize them according to your needs.
+You can add CSS classes from the base Twig which has its own `attr` object.
+For example, if you want to add custom CSS classes to quantity input in Add to Cart component, use the following:
+
+```html+twig
+{% set quantity_input_attr = {
+    class: 'ibexa-store-input ibexa-store-input--number ibexa-store-add-to-cart__quantity-input',
+} %}
+```
+
+Every element is also inside its own block so you can override the whole block.
+
+#### Extending JavaScript
+
+In case of the JavaScript component, you should import the original class and extend it:
+
+```js
+import Minicart from '@ibexa-cart/src/bundle/Resources/public/js/component/minicart';
+
+export default class StorefrontMinicart extends Minicart {}
+```
+
+The example below shows how to add a "Clear" button support to the maincart:
```

Comment:
We should probably mention, that this example does not cover twig part, which is also required.